packages/components/src/public/QueryModel/QueryModel.ts

```diff
@@ -627,6 +631,13 @@ export class QueryModel {
         return [...modelFilters, ...this.filterArray];
     }
 
+    get loadRowsFilters(): Filter.IFilter[] {
```

Comment:
To keep this logically consistent with `get filters()` consider:

```ts
get filters(): Filter.IFilter[] {
    return this.loadRowsFilters(false);
}

loadRowsFilters(excludeViewFilters?: boolean): Filter.IFilter[] {
    const modelFilters = this.getModelFilters(excludeViewFilters);
    if (this.keyValue !== undefined) return modelFilters;

    return [...modelFilters, ...this.filterArray];
}
```

then changing the call in `get loadRowsConfig()` to `filterArray: this.loadRowsFilters(true)`